pubspec.yaml

```diff
@@ -18,4 +18,4 @@ dependencies:
     sdk: flutter
 
 environment:
-    sdk: ">=1.19.0 <3.0.0"
+    sdk: ">=2.1.0 <3.0.0"
```

Comment:
Why do we need this?

Reply:
As of Dart 2.1, Future and Stream have been exported from dart:core. I'm adding tooling to catch these imports, and fixing them where I can.

Dart 2.1 was launched Nov 15, 2018, however, if you still want to support pre-Dart 2.1, it's totally acceptable to close this PR :)